test/test-starts-up-with-data-and-log-volumes

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset
+
+# This tests that neo4j starts up following the documented and recommended command:
+# docker run
+#   --publish=7474:7474 --publish=7687:7687
+#   --volume=$HOME/neo4j/data:/data
+#   --volume=$HOME/neo4j/logs:/logs
+#   neo4j
+
+. "$(dirname "$0")/helpers.sh"
+
+docker_run_with_data_and_logs_volumes() {
+  local l_image="$1" l_cname="$2" l_volumedir="$3"; shift; shift; shift
+
+  local envs=()
+  if [[ ! "$@" =~ "NEO4J_ACCEPT_LICENSE_AGREEMENT=no" ]]; then
+    envs+=("--env=NEO4J_ACCEPT_LICENSE_AGREEMENT=yes")
+  fi
+  for env in "$@"; do
+    envs+=("--env=${env}")
+  done
+  local cid
+  cid="$(docker run --detach "${envs[@]}" --name="${l_cname}" \
+  --publish=7474:7474 \
+  --publish=7687:7687 \
+  --volume="${l_volumedir}"/data:/data \
+  --volume="${l_volumedir}"/logs:/logs \
+  "${l_image}")"
+  echo "log: tmp/out/${cid}.log"
+  trap "docker_cleanup ${cid}" EXIT
+}
+
+readonly image="$1"
+readonly series="$2"
+readonly cname="neo4j-$(uuidgen)"
+
+readonly datadir=$(mktemp --directory)
```

Comment:
can you name this variable `data_and_logs_dir` or something that shows it contains both the data and the logs.

Reply:
done!